### ISSUE TITLE
Project doesn't compile (target iOS) with compose 1.2.0

### ISSUE THREAD

**Issue by fbattet** [JetBrains/compose-multiplatform#2441]
I have some problems with compose 1.3.0-alpha01-dev831 on iOS (#2440), so I decided to downgrade to version 1.2.0.
But the project doesn't compile anymore, I have got this error, when I target iOS : 

```
> Task :shared:iosSimulatorBuildIPhone13Debug FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':shared:iosSimulatorBuildIPhone13Debug'.
> External tool execution failed:
  * Command: [/usr/bin/xcrun, xcodebuild, -scheme, XXX, -project, ., -configuration, Debug, -derivedDataPath, build, -arch, x86_64, -sdk, iphonesimulator15.5]
  * Working dir: [...]
  * Exit code: 65
  * Standard output log: .../shared/build/compose/logs/iosSimulatorBuildIPhone13Debug/xcrun-2022-10-31-18-11-45-out.txt
  * Error log: .../shared/build/compose/logs/iosSimulatorBuildIPhone13Debug/xcrun-2022-10-31-18-11-45-err.txt
```

xcrun-2022-10-31-18-11-45-err.txt :
```
2022-10-31 18:11:48.261 xcodebuild[1307:15976] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
2022-10-31 18:11:48.262 xcodebuild[1307:15976] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
--- xcodebuild: WARNING: Using the first of multiple matching destinations:
{ platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder, name:Any iOS Simulator Device }
{ platform:iOS Simulator, id:7D8C9AFD-D524-453E-B6B0-812272A740C4, OS:15.5, name:device-IPhone8 }
{ platform:iOS Simulator, id:8574066A-045A-4EB4-B02A-E9B9562C4E21, OS:15.5, name:device-IPhone13 }
{ platform:iOS Simulator, id:3373FC59-587A-4076-866C-3BE34D3D01A3, OS:15.5, name:device-IPhone13_Pro_Max }
{ platform:iOS Simulator, id:9ACA35B9-19B7-4ABE-BD2D-042C97F16515, OS:15.5, name:iPad (9th generation) }
{ platform:iOS Simulator, id:9EEB6689-B10C-401A-A53A-90A79BA0D3F5, OS:15.5, name:iPad Air (5th generation) }
{ platform:iOS Simulator, id:FFD5AF13-8FBF-4000-B572-C18A87A03806, OS:15.5, name:iPad Pro (9.7-inch) }
{ platform:iOS Simulator, id:8A6161C4-AA5C-40E5-AF23-43A9740C7809, OS:15.5, name:iPad Pro (11-inch) (3rd generation) }
{ platform:iOS Simulator, id:133E8A5F-1AAE-41E9-954C-A4167C778F58, OS:15.5, name:iPad Pro (12.9-inch) (5th generation) }
{ platform:iOS Simulator, id:F8DD568B-4D98-474E-8F92-A939C1073DEC, OS:15.5, name:iPad mini (6th generation) }
{ platform:iOS Simulator, id:2759681A-DA87-426B-9379-186E5FB95A16, OS:15.5, name:iPhone 8 }
{ platform:iOS Simulator, id:0A9FAAAA-75DE-4DDE-9BBE-84FDAFB6A5B8, OS:15.5, name:iPhone 8 Plus }
{ platform:iOS Simulator, id:8BA895C8-92E3-457B-BDBF-7B6E57677C78, OS:15.5, name:iPhone 11 }
{ platform:iOS Simulator, id:097F82FF-59AA-40DF-B1E4-644469F9D8A9, OS:15.5, name:iPhone 11 Pro }
{ platform:iOS Simulator, id:0B20E50E-E152-4015-B9B1-307CE9D99A8B, OS:15.5, name:iPhone 11 Pro Max }
{ platform:iOS Simulator, id:A38DD610-523B-47D2-AD94-1212396CF221, OS:15.5, name:iPhone 12 }
{ platform:iOS Simulator, id:573E68CE-43E5-4286-9B14-C74E6F6FDF05, OS:15.5, name:iPhone 12 Pro }
{ platform:iOS Simulator, id:3AEC3569-AF5C-4636-82DA-9FB433A4470C, OS:15.5, name:iPhone 12 Pro Max }
{ platform:iOS Simulator, id:747A220A-87AA-4228-9A2C-6BFE88249F13, OS:15.5, name:iPhone 12 mini }
{ platform:iOS Simulator, id:54686BF5-5FBD-4425-9443-EFC3B7D5D47A, OS:15.5, name:iPhone 13 }
{ platform:iOS Simulator, id:E33382F8-63C1-4098-A59B-C1017C5FE381, OS:15.5, name:iPhone 13 Pro }
{ platform:iOS Simulator, id:58B947B2-93DB-4B0A-BE40-2AF5347024F5, OS:15.5, name:iPhone 13 Pro Max }
{ platform:iOS Simulator, id:12263A6D-9659-4C97-8065-B02E1909D562, OS:15.5, name:iPhone 13 mini }
{ platform:iOS Simulator, id:18A4BAE8-FEBB-4A57-A3C8-D128D110657E, OS:15.5, name:iPhone SE (3rd generation) }
{ platform:iOS Simulator, id:17C5A06A-A7D1-46E1-B834-16CC30C71C9F, OS:15.5, name:iPod touch (7th generation) }
{ platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Any iOS Device }
** BUILD FAILED **
```

**Comment by eymar** [comment]
Unfortunately, there is an issue in 1.2.0 with new xcode (`1.2.0` seems like it should be a stable version, but compose for ios is experimental(wip) and not guaranteed to behave correctly).

#2440 will be fixed and you'll be able to use 1.3.0-alpha*-dev* builds

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-2441">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
